src/components/views/article/Article.js

```diff
@@ -86,6 +90,22 @@ const Article = ({
               <ArticleComment />
               <div className="article__related__articles">
                 <h3>Related Articles</h3>
+                <div className="article__divider" />
+                <div className="related__article__container">
+                  {
+                  !isEmpty(relatedArticles) ? relatedArticles.map((related) => {
+                    if (related.id !== singleArticle.id) {
+                      return (
+                        <Card
+                          key={related.key}
```

Comment:
What is `related.key`?

Reply:
It's supposed to be `related.id`, I'll change it